aspnetcore/data/ef-rp/read-related-data.md

```diff
@@ -31,57 +32,87 @@ The following illustrations show the completed pages for this tutorial:
+
+* The `Select` operator loads only the related data needed, and is generally faster than using `Include`. [Loading related data with Select](#select) explains this in more detail.
 
 ### Performance considerations
 
-If you know you need related data for every entity retrieved, eager loading often offers the best performance, because a single query sent to the DB is typically more efficient than separate queries for each entity retrieved. For example, suppose that each department has ten related courses. Eager loading of all related data would result in just a single (join) query and a single round trip to the DB. A separate query for courses for each department would result in eleven round trips to the DB. The extra round trips to the DB are especially detrimental to performance when latency is high.
+If related data for every entity retrieved is needed:
 
-On the other hand, in some scenarios separate queries is more efficient. Eager loading of all related data in one query might cause a very complex join to be generated, which SQL Server can't process efficiently. Or if you need to access an entity's navigation properties only for a subset of a set of the entities you're processing, separate queries might perform better because eager loading of everything up front would retrieve more data than you need. If performance is critical, it's best to test performance both ways in order to make the best choice.
+* Eager loading generally offers the best performance.
+* Eager loading sends a single query to the DB. A single query is typically more efficient than separate queries for each entity retrieved.
 
-## Create a Courses page that displays Department name
+For example, suppose that each department has 10 related courses. Eager loading of all related data would result in just a single (join) query and a single round trip to the DB. A separate query for courses for each department would result in 11 round trips to the DB. The extra round trips to the DB are especially expensive when latency is high. Cloud environments can have high DB access latency.
 
-The Course entity includes a navigation property that contains the Department entity of the department that the course is assigned to. To display the name of the assigned department in a list of courses, you need to get the Name property from the Department entity that is in the `Course.Department` navigation property.
+In some scenarios, separate queries is more efficient. Eager loading of all related data might cause a complex join to be generated. SQL Server might not be able to process a complex join efficiently. If only a subset of related data needs to be accessed, separate queries might perform better. Eager loading loads all related data, and the extra data fetching may cost more than separate queries. As a general guideline, minimize round trips to the DB for best performance. Testing is required in order to find the optimal approach.
```

Comment:
This sounds like an argument based on EF Classic, rather than EF Core.

Reply:
@ajcvickers  it doesn't really fit into the article. I'm removing the entire paragraph.